File: beecrowd/1946.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	
	int n; 
	while(cin>>n){int div = 0;
	long long total = 1;
	n--;
	long long p2 = 1<<n;
	do{
		n--;
		div++;
		total = (total*n)/div;
	}while(n>div);
	total*=2;
	printf("%.2f\n", (100*total)/(double)p2);
	}
}

File: beecrowd/1946.py
while(True):
	try:
		n = int(input());
		n-=1;
		p = n;
		div = 1;
		total = 1.0;
		increment = 0;
		while(n>div):
			total = (total*n)/div;
			div+=1;
			n-=1;
			if(increment<=p):
				increment+=1;
				total/=2;
			
		print(total);
	except EOFError:
		break